python_ta/transforms/setendings.py

```diff
@@ -109,6 +113,38 @@
 ]
 
 
+class NodeDataStore():
+    """Collect data and log at end of all tests."""
+    def __init__(self):
+        """Store data without dupes."""
+        self._storage = set()
+
+        # Question: do we want to move logger to its own module, in a utils directory?
```

Comment:
I do not believe a class, nor a global variable, is necessary at all. It is fine to configure `logging` inside the main block of both the test file and the client.